src/aatoolbox/datasources/datasource.py

```diff
@@ -25,19 +27,27 @@ def __init__(
         country_config: CountryConfig,
         module_base_dir: str,
         is_public: bool = False,
+        is_global: bool = False,
     ):
 
         self._country_config = country_config
         self._module_base_dir = module_base_dir
         self._path_config = PathConfig()
         self._raw_base_dir = self._get_base_dir(
-            is_public=is_public, is_raw=True
+            is_public=is_public,
+            is_global=is_global,
```

Comment:
I wouldn't necessarily make this a instance parameter, but rather just a parameter that you can pass to `_get_base_dir` that can be used in the class itself (you'd have to just call `_get_base_dir` again). I'd imagine that you probably only want to use it for the raw data, then you'd want to keep the country iso3 for the processed data.